docs/IE_DG/supported_plugins/GNA.md

```diff
@@ -159,6 +162,29 @@ Heterogeneous plugin was tested with the Intel&reg; GNA as a primary device and
 
 > **NOTE:** Due to limitation of the Intel&reg; GNA backend library, heterogenous support is limited to cases where in the resulted sliced graph, only one subgraph is scheduled to run on GNA\_HW or GNA\_SW devices.
 
+## Support for Quality of Service (QoS) Mechanism on Windows\*
+
+As noted in the introduction, GNA is designed for real-time workloads such as noise reduction or speech recognition. 
+For such workloads, processing should be time constrained, otherwise extra delays may cause undesired effects such as
+audio "glitches". To make sure that processing can satisfy real time requirements, the GNA driver provides a QoS
+mechanism which interrupts the requests which cannot be fulfilled in time.
+
+When an inference (in `GNA_HW` mode) cannot be executed within the time frame specified by the timeout defined in
+Windows\* registry settings, then `InferRequest::Wait()` will return status code
```

Comment:
I would not mention the registry.  The focus here should be on ensuring that your application runs smoothly given the possibility of interruption by high priority Windows audio tasks.

Reply:
Fixed (I hope)